### DISCUSSION TITLE
Inquiry Regarding OpenSSL Support in OpenSCAP

### DISCUSSION THREAD

**Discussion by samirMhn** [OpenSCAP/openscap#2125] [Q&A]
Hello,

I am writing to inquire about the future plans for OpenSCAP, specifically regarding the addition of OpenSSL support. Currently, OpenSCAP supports NSS3 and libgcrypt, but I am interested in knowing if there are any plans to include support for OpenSSL in upcoming releases.

We are currently using OpenSSL as the only FIPS-certified crypto module, and I would like to know if there are any plans to add support for libopenssl as a crypto module for OpenSCAP.

Thank you for your time and for the continuous development of OpenSCAP.

We currently are using openssl as the only FIPS certified crypto module and I would like to know if there is a plan to add support of libopenssl ad crypto for openscap

Thank you for your time and for the continuous development of OpenSCAP.

Sam M

**Comment by evgenyz** [comment]
Not at the moment. GCrypt goes away in 1.4, though.

**Reply by samirMhn** [reply]
Does this mean that in version 1.4 there will be no crypto support at all, or will gcrypt be removed and only NSS3 kept?

**Reply by evgenyz** [reply]
NSS/GCrypt is used by the scanner only for hash calculations. Starting from 1.4 it'd be NSS only. OpenSSL is used indirectly for XML signature validation.
